packages/tocco-ui/src/EditableValue/typeEditors/TimeEdit.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import StyledTimeEdit from './StyledTimeEdit'
+import Button from '../../Button'
+import {stringToDuration, numbersToTimeFormat} from '../utils'
+
+const TimeEdit = props => {
+  const hours = props.value.hourOfDay
+  const minutes = props.value.minuteOfHour
+
+  const timeString = numbersToTimeFormat(hours, minutes)
+
+  const handleChange = e => {
+    if (props.onChange) {
+      props.onChange(stringToDuration(e.target.value))
+    }
+  }
+
+  const clearInput = () => {
+    const emptyTimeObject = {hoursOfDay: undefined, minuteOfHour: undefined}
+    if (props.onChange) {
+      props.onChange(emptyTimeObject)
+    }
+  }
+
+  return (
+    <StyledTimeEdit>
+      <input
+        className="form-control"
+        type="time"
+        value={timeString}
+        onChange={handleChange}
+        name={props.name}
+        id={props.id}
+        readOnly={props.readOnly}
+      />
+      <Button
+        icon="times"
+        look="raised"
+        onClick={clearInput}
+      />
+    </StyledTimeEdit>
```

Comment:
TimeEdit works well, but when i enter 12 in the hours field and press tab twice to leave the field i expect it to set 00 into minutes field. or at least that its validated properly. rightnow i get de fault html validation if i try to submit the form with 12:--